packages/handlersjs-logging/lib/loggers/pino-logger.ts

```diff
@@ -39,69 +71,35 @@ export class PinoLogger extends Logger {
 
     if (level <= this.minimumLevel) {
 
-      const logData = level > this.minimumLevelPrintData
-        ? {}
-        // if data is a string, convert it to an object with a data property. Pino expects data to be an object. If it remains a string, it will use the data as a message, and drop the rest of our log.
-        : typeof data === 'string'
-          ? { data }
-          // if data is undefined, convert it to an empty object.
-          : data ?? {};
+      const parsedData = level > this.minimumLevelPrintData ? {} : data ?? {};
 
+      const logData = { variables: this.variables, data: parsedData };
```

Comment:
before when it was an empty string it was turned into an empty object, which pino knew not to log. Now however, since we are always logging some data with the variables, I think an empty string will cause it to be logged as
```
data: {},
```
Can you confirm that it results in the empty object being logged? Or does Pino simply not log the empty data key?

If it logs it as an empty object, perhaps we should convert empty strings to undefined instead?

Reply:
Changed it so the data object is only logged when it is defined or not an empty string.